wpilibj/src/main/java/edu/wpi/first/wpilibj/util/Units.java

```diff
@@ -0,0 +1,104 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.util;
+
+/**
+ * Utility class that converts between commonly used units in FRC.
+ */
+public final class Units {
+  private static final double kInchesInAFoot = 12.0;
+  private static final double kMetersInAInch = 0.0254;
+  private static final double kHalfAMinuteInSeconds = 30;
```

Comment:
I would make this just be `kSecondsPerMinute` and divide by 2 in the math. This is so the maintenance factor is lower.